pkg/cmd/server/origin/legacyconfigprocessing/webconsole_proxy.go

```diff
@@ -0,0 +1,164 @@
+package legacyconfigprocessing
```

Comment:
the whole package is duplicated code, right? nothing to review.

Reply:
> the whole package is duplicated code, right? nothing to review.

correct